### ISSUE TITLE
Guest author format

### ISSUE THREAD

**Issue by cjyabraham** [cncf/cncf.io#205]
Normally in the byline is says "By Kim McMahon".  When a guest author is set it omits "By" and just puts the name.  Should we keep the "By" present?  And the guest author currently doesn't show in the [blog listing](https://dev-cncfci.pantheonsite.io/blog/).  Can we have it there too?

**Comment by thetwopct** [comment]
I removed the "By" because the guest author doesn't always follow the same pattern. For example, over the last month we have some of the following:

- Guest post originally published on the Alluxio Engineering Blog by Rong Gu
- Guest post originally published on the Rookout blog by Liran Haimovitch, co-founder and CTO of Rookout
- Guest post by Joe Pelletier, VP of Strategy at Fairwinds
- Guest post by Chad Serino, CEO, AlphaBravo
- Guest Post originally published on Medium by Saurabh Gupta, Sr. Developer Advocate @DigitalOcean
- Guest blog post from Maxime Guyot Kubespray
- Originally published on goharbor.io by Alex Xu Harbor Contributor and Senior Product Manager, VMware

Adding "By" in front of any of these doesn't make much sense to me.

Adding this line to the blog archives doesn't look good due to the lengths of these and the limited space, so we decided not to add them until we know more from the Editors.

A nicer solution could be that we add 2 meta fields, one just for the guest author name, and one for the guest author name and description. But without more feedback from the Editors I would be reluctant to proceed.
